DataFormats/ForwardDetId/interface/HGCalTriggerModuleDetId.h

```diff
@@ -94,7 +94,7 @@ class HGCalTriggerModuleDetId : public DetId {
   static const int kHGCalLayerMask = 0x1F;
   static const int kHGCalTypeOffset = 19;
   static const int kHGCalTypeMask = 0x3;
-  static const int kHGCalZsideOffset = 24;
+  static const int kHGCalZsideOffset = 21;
```

Comment:
This change is already included in the dev branch (but your commit has been rebased on the dev branch, which is probably why it appears again here as a different commit).
Could you rebase (or cherry-pick) just the second commit on top of the dev branch?

Reply:
Done, sorry about that